src/postprocess/band_structure.jl

```diff
@@ -19,18 +18,32 @@ function high_symmetry_kpath(model; kline_density=20)
     # https://github.com/louisponet/DFControl.jl/blob/master/src/structure.jl
     # but for this the best way to go would be to refactor into a small "CrystalStructure"
     # julia module which deals with these sort of low-level details everyone can agree on.
-    pystructure = pymatgen_structure(model.lattice, model.atoms)
-    symm_kpath = pyimport("pymatgen.symmetry.bandstructure").HighSymmKpath(pystructure)
-    kcoords, labels = symm_kpath.get_kpoints(kline_density, coords_are_cartesian=false)
+
+    # setup for Spglib.jl
+    # tranpose our lattice to match spglib convention
+    spg_positions, spg_numbers, _ = spglib_atoms(model.atoms)
+    structure = Cell(transpose(model.lattice), spg_positions, spg_numbers)
+    # get standardized cell and spacegroup number for Brillouin.jl 
+    spg_lattice = standardize_cell(structure, to_primitive=false).lattice
+    # comparge spglibe primitive to DFTK primitive
+    spg_primitive = standardize_cell(structure, to_primitive=true).lattice
+    spg_primitive ≈ transpose(model.lattice) || @warn "The DFTK lattice and spglib's primitive lattice do not agree."
+   
+    # get spacegroup num from spglib dataset
+    sgnum = get_spacegroup_number(structure)
+    
+    Rs = [spg_lattice[i, :] for i in 1:size(spg_lattice,1)]
```

Comment:
Careful here: I think spglib uses lattice vectors as rows and not as columns (as we do in DFTK). Since all this conversion between different conventions gets pretty messy, I would suggest to do the wrapping of `Spglib.jl` to match DFTK conventions also in `spglib.jl`, where we actually already have a  `spglib_standardize_cell`  that you can adapt. 

Eventually we will replace all calls of DFTK to spglib_jll by calls to `Spglib.jl` so I think it would be good to keep all calls to spglib at a central place for now.

Reply:
I think Brillouin expects Rs to be a list of lattice vectors. This is why I create this from the spg_lattice, which is already formatted in rows. I will double check everything and move this into external/spglib.jl library so that everything that happens there and cleans up the band structure file.